index.js

```diff
@@ -1,7 +1,53 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Julio";
+console.log("The driver's name is " + hacker1);
+
+let hacker2 = "Christian";
```

Comment:
```suggestion
const hacker2 = "Cristian";
```

Siempre que puedas const (lo del nombre es coña, pero es que se escribe sin la h jejeje xD)

Reply:
Cierto!! Cont const!! Y lo de Christian es por que el de mi hermano se escribe con h....deberiais juntaros todos los Chri/Cristians del mundo y aclararos.